File: static/simp.js
// static/simp.js

let simpFunctionCalled = false;

function addNumbers() {
    let num1 = document.getElementById("num1").value;
    let num2 = document.getElementById("num2").value;

    axios.get(`/simp/add/${num1}/${num2}`)
        .then(response => {
            document.getElementById("result").innerText = `Result: ${response.data.result}`;
            simpFunctionCalled = true;
        })
        .catch(error => {
            console.error("Error:", error);
        });
}

function subtractNumbers() {
    let num1 = document.getElementById("num1").value;
    let num2 = document.getElementById("num2").value;

    axios.get(`/simp/subtract/${num1}/${num2}`)
        .then(response => {
            document.getElementById("result").innerText = `Result: ${response.data.result}`;
            simpFunctionCalled = true;
        })
        .catch(error => {
            console.error("Error:", error);
        });
}

File: simp/simp.py
from flask import Blueprint, jsonify

simp_bp = Blueprint("simp", __name__)

@simp_bp.route("/add/<int:num1>/<int:num2>", methods=["GET"])
def add_numbers(num1, num2):
    result = num1 + num2
    return jsonify({"result": result})

@simp_bp.route("/subtract/<int:num1>/<int:num2>", methods=["GET"])
def subtract_numbers(num1, num2):
    result = num1 - num2
    return jsonify({"result": result})
